src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -159,78 +159,103 @@ class NetworkExpressRouteScenarioTest(ResourceGroupVCRTestBase):
-            self.resource_group, self.express_route_name), checks=NoneCheck())
-        # Expecting no results as we just deleted the only express route in the resource group
-        self.cmd('network express-route circuit list --resource-group {}'.format(self.resource_group),
-            checks=NoneCheck())
+        self.cmd('network express-route get-stats --resource-group {} --name {}'.format(rg, circuit),
+            checks=JMESPathCheck('type(@)', 'object'))
 
-class NetworkExpressRouteCircuitScenarioTest(ResourceGroupVCRTestBase):
+        self.cmd('network express-route update -g {} -n {} --set tags.test=Test'.format(rg, circuit),
+            checks=JMESPathCheck('tags', {'test': 'Test'}))
 
-    def __init__(self, test_method):
-         # TODO The resources for this test did not exist so the commands will return 404 errors.
-         # So this test is for the command execution itself.
-        super(NetworkExpressRouteCircuitScenarioTest, self).__init__(__file__, test_method)
-        self.resource_group = 'express_route_circuit_scenario'
-        self.express_route_name = 'test_route'
-        self.placeholder_value = 'none_existent'
-
-    def test_network_express_route_circuit(self):
-        self.execute()
+        self._test_express_route_auth()
 
-    def body(self):
-        rg = self.resource_group
-        ern = self.express_route_name
-        pv = self.placeholder_value
-        allowed_exceptions = "The Resource 'Microsoft.Network/expressRouteCircuits/{}' under resource group '{}' was not found.".format(ern, rg)
-        self.cmd('network express-route circuit-auth show --resource-group {0} --circuit-name {1} --name {2}'.format(
-            rg, ern, pv), allowed_exceptions=allowed_exceptions)
-        self.cmd('network express-route circuit-auth delete --resource-group {0} --circuit-name {1} --name {2}'.format(
-            rg, ern, pv), allowed_exceptions=allowed_exceptions)
-        self.cmd('network express-route circuit-peering delete --resource-group {0} --circuit-name {1} --name {2}'.format(
-            rg, ern, pv), allowed_exceptions=allowed_exceptions)
-        self.cmd('network express-route circuit-peering show --resource-group {0} --circuit-name {1} --name {2}'.format(
-            rg, ern, pv), allowed_exceptions=allowed_exceptions)
-        self.cmd('network express-route circuit list-arp-tables --resource-group {0} --name {1} --peering-name {2} --device-path {2}'.format(
-            rg, ern, pv), allowed_exceptions=allowed_exceptions)
-        self.cmd('network express-route circuit list-route-tables --resource-group {0} --name {1} --peering-name {2} --device-path {2}'.format(
-            rg, ern, pv), allowed_exceptions=allowed_exceptions)
+        self._test_express_route_peering()
+
+        self.cmd('network express-route list-arp-tables --resource-group {} --name {} --peering-name azureprivatepeering --path primary'.format(rg, circuit))
```

Comment:
I didn't add validation here because these don't return anything. The circuit never actually gets created so these tables never get created. Thus it is really just testing that the request makes it through the SDK without error. I'll add a comment though.